docs/src/modules/components/PickersPlayground.tsx

```diff
@@ -592,8 +593,8 @@ export default function PickersPlayground() {
             </DemoContainer>
           )}
         </ComponentSection>
-        <Divider orientation="vertical" light sx={{ display: { xs: 'none', sm: 'flex' } }} />
-        <Divider light sx={{ display: { xs: 'auto', sm: 'none' } }} />
+        <Divider orientation="vertical" light sx={{ display: { xs: 'none', md: 'flex' } }} />
```

Comment:
Align layout change trigger to `md` in order to fix broken layout between `sm` and `md`.